sphinx_gallery/tests/test_binder.py

```diff
@@ -95,11 +91,10 @@ def apptmp():
         assert iconf == {}
 
     # Assert extra unkonwn params
-    with pytest.raises(ValueError) as excinfo:
-        conf7 = deepcopy(conf1)
-        conf7['foo'] = 'blah'
+    conf7 = deepcopy(conf1)
+    conf7['foo'] = 'blah'
+    with pytest.raises(ValueError, match='Unknown Binder config key'):
```

Comment:
I had no idea about this

![](https://media.giphy.com/media/3og0IMJcSI8p6hYQXS/giphy.gif)